graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -20,19 +21,41 @@
 )
 
 
-def merge_schemas(schema_id_to_ast):
-    """Check that input schemas do not contain conflicting definitions, then merge.
+CrossSchemaEdgeDescriptor = namedtuple(
+    'CrossSchemaEdgeDescriptor', (
+        'edge_name',  # str, name used for the corresponding in and out fields
+        'outbound_side',  # FieldReference for the outbound (source) field
+        'inbound_side',  # FieldReference for the inbound (sink) field
+        'out_edge_only',  # bool, defaults to False, whether to not add the in-edge
+    )
+)
+
+
+CrossSchemaEdgeDescriptor.__new__.__defaults__ = (False,)
```

Comment:
I am against the choice of having out_edge_only default to False. I am in favor of user the having the choice between building a one-directional vs bidirectional edge through the out_edge_only parameter. However, I think that it important that this choice is as explicit as possible so that the user is conscious of what he or she is doing because adding unintended edges seems like a very plausible and detrimental bug.

Reply:
That's a good point, having the default value makes it less obvious that there is a choice. I'll make that change.